src/overlays/actors/ovl_Bg_Ganon_Otyuka/z_bg_ganon_otyuka.h

```diff
@@ -4,14 +4,34 @@
 #include "ultra64.h"
 #include "global.h"
 
+#define SIDE_EAST (1 << 0)
+#define SIDE_WEST (1 << 1)
+#define SIDE_SOUTH (1 << 2)
+#define SIDE_NORTH (1 << 3)
+#define SIDE_ALL (SIDE_EAST | SIDE_WEST | SIDE_SOUTH | SIDE_NORTH)
```

Comment:
Probably a good idea to add a prefix specific to the actor for those, to avoid any conflict with the rest of the codebase, especially since they are exposed in the header.

Also if they are only meant for use inside the actor itself, they shouldn't be exposed in the header. Same goes for the FlashState enum.

Reply:
I know that `SIDE_ALL` is used in Ganondorf when he makes platforms fall though I'm not sure whether or not flash state is modified outside the actor. I'd assume not, so I could move flash state to the c file if that'd be better.

Reply:
Alright, that seems good to me.